CHANGELOG.rst

```diff
@@ -52,6 +52,9 @@ Changed
   to Docker Registry defined in the GitLab service is properly applied during
   installation/management.
 
+- [debops.phpipam] The relevant inventory variables have been renamed, check the
+  :ref:`upgrade_notes` for details.
```

Comment:
You should also mention that the role now uses the upstream phpIPAM repository, and the version bump to 1.3.2.

Reply:
Done